extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerDeploymentTemplate.java

```diff
@@ -39,7 +39,9 @@ public void registerTracer(JaegerConfig jaeger) {
         if (!registered) {
             if (isValidConfig(jaeger)) {
                 initTracerConfig(jaeger);
-                GlobalTracer.register(new QuarkusJaegerTracer());
+                QuarkusJaegerTracer quarkusJaegerTracer = new QuarkusJaegerTracer();
+                log.infof("Registering tracer to GlobalTracer %s", quarkusJaegerTracer);
```

Comment:
Does it have to be an "info" log?
Wondering if "debug" would not be more appropriate.

Reply:
I would leave it as info - it often helps to investigate issues when people complain why Jaeger is not receiving spans.

Reply:
@pavolloffay  Well, you said it, it's useful when investigating issues so it should be debug, not info (for the context, we try to minimize the log output of Quarkus to what is really important). Could you change that and squash?

Other than that, it looks good to go.

Thanks!

Reply:
By doing so one has to redeploy and change config. Ok I will change that.

Reply:
Sure. But if we have one line per component, we end up with a startup with 20 log lines announcing things are started/configured.

The bar is really high (or it's hard to disable it) to have a log line coming from a component. Typically, it's better to have warnings if the configuration is not OK, and that's what it already does.

Reply:
changed